subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -37,6 +37,16 @@ extern "C" {
  */
 #define EXECUTION_OBJ_DECODED_BIT 2
 
+/** @brief AWS FOTA JSON parse result. */
+enum aws_fota_json_result {
+	AWS_FOTA_JSON_RES_SKIPPED = 1,		/*!< Not a FOTA job document, skipped */
```

Comment:
```suggestion
	AWS_FOTA_JSON_RES_SKIPPED = 1,		/* Not a FOTA job document, skipped */
```

Reply:
Nit: Not familiar with this formatting, what value does it add?

Reply:
And why doesn't success start at 0 and the rest including `AWS_FOTA_JSON_RES_SKIPPED` in decreasing order? Why have both negative and positive values

Reply:
> Nit: Not familiar with this formatting, what value does it add?

That's doxygen syntax for putting documentation after members: https://www.doxygen.nl/manual/docblocks.html#memberdoc



> And why doesn't success start at 0 and the rest including `AWS_FOTA_JSON_RES_SKIPPED` in decreasing order? Why have both negative and positive values

Because `AWS_FOTA_JSON_RES_SKIPPED` is not an error. This mimics the existing implementation and the way it is done in `aws_fota_mqtt_evt_handler`.